internal/pkg/config/config.go

```diff
@@ -195,6 +209,10 @@ func redactServer(cfg *Config) Server {
 		redacted.Instrumentation.SecretToken = kRedacted
 	}
 
+	for i := range redacted.StaticPolicyTokens.PolicyTokens {
+		redacted.StaticPolicyTokens.PolicyTokens[i].TokenKey = kRedacted
```

Comment:
There's currently a test failure with static tokens; i think this is occuring because `redacted` is the same `cfg` object that is checked when an auth token is recieved; i'll add a `Copy` func for server and outputs so we don't accidently erase real values.